### DISCUSSION TITLE
configure t-pot as mysql server

### DISCUSSION THREAD

**Discussion by reemoove** [telekom-security/tpotce#1123] [Q&A]
can I configure the t-pot to be mysql server or ssh or what I want ?? can I put it directly after installation in the network or there something to do??

**Comment by e101sg** [comment]
Dear Community, 
any suggestion for configuring Mysql server. I am trying... please share any suggestion. Thanks & Cheers!

**Reply by e101sg** [reply]
I have installed the MySql server as a part of the DAMN VULNERABLE WEB APPLICATION (https://github.com/digininja/DVWA).
MySql is worked part of the DVWA. Need to change few things in Database setup in DVWA. 

No sure, how to setup mysql server for T-pot or is it necessary since the logs are processed to ELK stack. Please correct me, in case i am wrong.  Thanks.
